client/landing/jetpack-cloud/sections/backups/index.js

```diff
@@ -2,11 +2,13 @@
  * External dependencies
  */
 import page from 'page';
+import * as FullStory from '@fullstory/browser';
```

Comment:
Recommend switching to something like:

```suggestion
import { init as initFullStory } from '@fullstory/browser';
```

This makes it clearer exactly what's being imported and used right at the top of the file. This suggestion applies to all files.